tcms/static/js/testrun_actions.js

```diff
@@ -167,7 +167,7 @@ Nitrate.TestRuns.Details.on_load = function() {
     delCaseRun(jQ(this).data('param'));
   });
   jQ('.js-update-case').bind('click', function() {
-    postToURL(jQ(this).data('param'), serializeCaseRunFromInputList('id_table_cases', 'case_run'));
+    updateCaseRunText();
   });
```

Comment:
this handler used to be a generic one, attached to a lot buttons (hence the generic selector - `.js-update-case`)
all of this buttons are now gone and refactored to use jsonRPC probably with other handlers (`grep js-update-case` yields only one result). So it's better to use a move-specific selector here. 
https://github.com/kiwitcms/Kiwi/pull/1549/files#diff-13f4b36666bb289a60fcaf7fd8630ef5R26 - the element we care about has id=`update_case_run_text`, so let's use that instead of `.js-update-case`